pagopa/io-app
# chore(IT Wallet): [SIW-1356] Update ToS and Privacy url

## Short description
This PR updates Privacy and ToS url in discovery info screen and identification

## List of changes proposed in this pull request
- Adds privacy and tos url

## How to test
Start the app and check if in discovery info screen the privacy and tos url at the bottom works correctly. Also check the privacy url in identification mode selection.
